vocode/streaming/transcriber/deepgram_transcriber.py

```diff
@@ -156,21 +227,74 @@ async def _run_loop(self):
             except Exception as e:
                 self.logger.warning(f"VAD error: {str(e)}", exc_info=True)
 
-    def send_audio(self, chunk):
-        if not self.transcriber.is_muted:
-            self.consume_nonblocking(chunk)
-        else:
-            self.consume_nonblocking(
-                {
-                    "chunk": b"",
-                    "ignore": True,
-                    "timestamp": time_ns(),
-                    "is_silence": True,
-                }
-            )
+    def is_volume_low(self, chunk: bytes) -> bool:
```

Comment:
github has formatted this change in an unreadable way. Are there any functionality changes to the existing path?